releaseNewVersion.sh

```diff
@@ -74,6 +74,8 @@ echo -e "\n"
 perl -pi -e "s/\"version\": \"${packagesVersion}\"/\"version\": \"${newVersion}\"/" package.json
 perl -pi -e "s/version = \"${srcVersion}\";/version = \"${newVersion}\";/" src/version.js
 unset packagesVersion srcVersion parts bumps yn
+# apply version changes to the NPM lockfile
+npm install --ignore-scripts
```

Comment:
Hmm. This will automatically bump dependencies (within the allowed range) which may result in the released version differing from the version we tested and "signed off on". I do like the idea of updating dependencies, but not sure this is the way to go about it...

Reply:
Changed – see cf66d0332653ff2ec133dbe8c7b951d9fac7abd6

Reply:
Ahh - I interpreted the intention here as version changes (updates) to dependencies, not version changes of auspice!

Reply:
We should update dependencies straight _after_ the release so we have it tested for the next one